frontend/annotation_resolver.py

```diff
@@ -65,27 +69,27 @@ def resolve(self, annotation):
         if annotation[:4] == "List":
```

Comment:
Oh, okay. But... why do you unparse them first and then work with the unparsed string here? Is that easier than just using the AST node here?